docsets/RxSwift/docset.json

```diff
@@ -1,13 +1,17 @@
 {
     "name": "RxSwift",
-    "version": "4.4.0",
+    "version": "6.5.0",
     "archive": "RxSwift.tgz",
     "author": {
         "name": "Petr Zvonicek",
         "link": "https://twitter.com/zvonicek"
     },
     "aliases": [],
     "specific_versions": [
+        { 
+            "version": "6.5.0",
+            "archive": "versions/6.5.0/RxSwift.tgz"
```

Comment:
You need to add the archive at this path as well. I'll add it by copying the `docsets/RxSwift/RxSwift.tgz`. If this is not ok, please let me know.

Reply:
It's my forgotten thing. Sorry!